float32 rumble_duration
float32 rumble_value
